sql/alps_submitter.go

```diff
@@ -0,0 +1,167 @@
+package sql
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+)
+
+type GraphConfig struct {
```

Comment:
Should we expose `GraphConfig` struct? If no, turn to `graphConfig` with lowercase.

Reply:
Fixed